crates/wash-cli/Cargo.toml

```diff
@@ -59,8 +59,9 @@ warp = { workspace = true }
 warp-embed = { workspace = true }
 wascap = { workspace = true }
 wash-lib = { workspace = true, features = ["cli", "parser", "nats", "start"] }
-wasmbus-rpc = { workspace = true }
 wasmcloud-control-interface = { workspace = true }
+wasmcloud-core = { workspace = true }
+wasmcloud-provider-sdk = { workspace = true }
```

Comment:
This is okay for now, but I don't love bringing in this dependency. It looks like it's just for `wash call`, which doesn't currently work anyway, so we could also leave this out for now

Reply:
I don't feel like removing functionality is in-scope for this PR, let's keep functional changes for a different PR. This is just removing the problematic and unmaintained `wasmbus-rpc`